### PR TITLE
Implemented age_tail function

### PR DESCRIPTION
**Pull request by MatheusFarias03**
age_tail() function returns a list containing all the elements, excluding the first one, from a list.

### CODE REVIEW

**Review by rafsun42 — APPROVED** [review]
Looks good. Did  you mean to send this PR to master?

File: src/backend/utils/adt/agtype.c [hunk 1]
```diff
@@ -4994,6 +4994,84 @@ Datum age_last(PG_FUNCTION_ARGS)
...
+    /* check for null */
+    if (PG_ARGISNULL(0))
+    {
+        PG_RETURN_NULL();
+    }
+
+    agt_arg = AG_GET_ARG_AGTYPE_P(0);
+    /* check for an array */
+    if (!AGT_ROOT_IS_ARRAY(agt_arg) || AGT_ROOT_IS_SCALAR(agt_arg))
+
```
Comment (jrgemignani) [inline src/backend/utils/adt/agtype.c:5036]: extra line

**Review by jrgemignani — CHANGES_REQUESTED** [review]

**Review by jrgemignani — APPROVED** [review]

File: src/backend/utils/adt/agtype.c (hunk 1 above)
Comment (jrgemignani) [reply src/backend/utils/adt/agtype.c]: Still appears to have an extra newline

**Review by jrgemignani — CHANGES_REQUESTED** [review]

**Comment by MatheusFarias03** [comment]
Yeah quite odd actually, I did though remove on one of the commits as you can see, but it still shows the extra line... I'll check again soon.

**Review by jrgemignani — APPROVED** [review]

**Comment by jrgemignani** [comment]
@MatheusFarias03 ty! merged

**Pull request merged**